content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+
+---
+title: "Accountability and fault detection"
+linkTitle: "Accountability and fault detection"
+weight: 6
+description: >
+  Autonity's Accountability Fault Detection model, reporting, proving innocence or guilt, temporal constraints, and economics for reporting offences and penalties for Byzantine behaviour.
+---
+
+
+## Overview
+
+This section describes the Autonity accountability and fault detection protocol, the role of validators in submitting and verifying accountability event proofs (fault, accusation, innocence), the lifecycle for fault accountability and detection, and slashing.
+
+Autonity implements an _accountability and fault detection_ (AFD) protocol for detecting infractions of consensus rules by validators participating in [consensus](/glossary/#consensus) as [consensus committee](/glossary/#consensus-committee) members. [Consensus](/glossary/#consensus) rules govern committee behaviour during consensus while participating in block proposal and voting as a committee member. Failure to adhere to these rules is a _rule infraction_ and AFD will detect and apply slashing penalties for proven rule infractions.
+
+Proven rule infractions are reported as _faults_ and slashing makes faults _accountable_. Faults are _detected_ by validators and submitted on chain as accountability _events_ providing proof of behaviour. This proof is derived from consensus messaging between validators during consensus rounds and submitted on-chain to the Accountability Contract.
+
+It is important to note that AFD runs alongside Autonity's Tendermint proof of stake consensus implementation and is _fully automated_: accountability events are generated and processed by protocol; no manual intervention by validator operators is required. 
+
+AFD functions by submitting, verifying, and processing accountability event proofs by epoch. Suspected rule infractions are:
+
+- directly submitted as a _fault_ proof by a _reporting_ validator
+- promoted from accusations where they are:
+  - reported as an _accusation_ proof, submitted by a _reporting_ validator against an _offending_ validator
```

Comment:
```suggestion
  - reported as an _accusation_, submitted by a _reporting_ validator against an _offending_ validator
```

I wouldn't call accusation proofs. They are accountability events but they are not proofs

Reply:
OK, clear.